multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -53,7 +53,7 @@ coreos:
         WantedBy=multi-user.target
 
     - name: decrypt-tls-assets.service
-      enable: {{ne .KMSKeyARN ""}}
+      enable: true
```

Comment:
`enable: true` allows the `Install` section to be evaluated. That's how it registers as a dependency of `kubelet.service`